examples/pipelines/finetune_stable_diffusion/README.md

```diff
@@ -0,0 +1,131 @@
+
+
+### Examples
+If you want to test out Stable Diffusion, you can use the following demos:
+
+* [RunwayML](https://runwayml.com/ai-magic-tools/text-to-image/)
+* [Stable Diffusion v2.1 - Hugging Face space](https://huggingface.co/spaces/stabilityai/stable-diffusion)
+* [Stable Diffusion v1.5 - Hugging Face space](https://huggingface.co/spaces/runwayml/stable-diffusion-v1-5)
+
+## Pipeline Overview
+
+The image below shows the entire pipeline and its workflow. Note that this workflow is currently adapted to the interior design domain, but can be easily adapted to other domains by changing the prompt generation component.
+
+![Image](../../../docs/art/stable_diffusion/stable-diffusion-pipeline.png)
+
+
+There are 5 components in total, these are:
+
+1. **Load from hub**: This component loads a dataset from the Hugging Face dataset hub. It takes in a Hugging Face dataset ID, so it can use any [Hugging Face dataset](https://huggingface.co/datasets) (or a custom one).
+
+2. **Image embedding**: This component takes in the images of the dataset and returns the CLIP embeddings of these images. These embeddings can be used for filtering the dataset or for retrieving similar images from the LAION-5B dataset.
+
+3. **Image URL Retrieval**: This component retrieves images from the [LAION-5B](https://laion.ai/blog/laion-5b/) dataset based on the seed prompts. The retrieval itself is done based on CLIP embeddings similarity between the prompt embedding and the visual embeddings in the LAION dataset. This component doesn’t return the actual images yet, only the URLs. The next component in the pipeline will then download these images.
+
+4. **Download Images**: This component downloads the actual images based on the URLs retrieved by the previous component. It takes in the URLs as input and returns the actual images, along with some metadata (like their height and width).
+
+5. **Add Captions**: This component captions all images using [BLIP](https://huggingface.co/docs/transformers/model_doc/blip). This model takes in the image and generates a caption that describes the content of the image. This component takes in a Hugging Face model ID, so it can use any [Hugging Face Hub model](https://huggingface.co/models).
+
+
+
+## Install and Run
+
+### Requirements
+
+```
+pip install git+https://github.com/ml6team/fondant.git
+pip install fondant[pipelines]
+```
+
+### Pipeline Configuration
+
+The pipeline_configs.py contains a data class used to store two general configuration parameters for the pipeline.
+
+- ```BASE_PATH```: This base path used to store the artifacts 
+- ```HOST```: This is the Kubeflow pipelines host url
+
+Both of these need to be set to suitable values.
+
+
+### Run
+
+Running the pipeline then consists of two steps:
+
+1. Building the images for each of the pipeline components
+```
+bash build_images.sh --namespace <NAMESPACE> --repo <REPO> -C all
```

Comment:
I usually run the following in my terminal:
```suggestion
sh build_images.sh --component all
```

Reply:
This won't work for users though, as they can't push to our registry, which is the default.

Reply:
So leave it as it is?